tests/unit_tests/test_tethys_portal/test_middleware.py

```diff
@@ -22,13 +22,14 @@ def tearDown(self):
     def test_process_exception_anonymous_user(
         self, mock_redirect, mock_hasattr, mock_isinstance
     ):
+        mock_get_response = mock.MagicMock()
         mock_request = mock.MagicMock()
         mock_exception = mock.MagicMock()
         mock_hasattr.return_value = True
         mock_isinstance.return_value = True
         mock_request.user.is_anonymous = True
 
-        obj = TethysSocialAuthExceptionMiddleware()
+        obj = TethysSocialAuthExceptionMiddleware(mock_get_response)
```

Comment:
Is there a corresponding change we need to make in the `TethysSocialAuthExceptionMiddleware`?

Reply:
No, as far as I can tell it was already compatible with the newer way of calling it (passing in the get_response). The latest change just dropped support for the old way of calling it (without the get_response arg). The only change necessary was to update our tests to use the new style.